var/spack/repos/builtin/packages/brion/package.py

```diff
@@ -22,6 +22,7 @@ class Brion(CMakePackage):
     version('3.3.1', tag='3.3.1', submodules=True)
     version('3.3.2', tag='3.3.2', submodules=True)
     version('3.3.3', tag='3.3.3', submodules=True)
+    version('3.3.4', tag='3.3.4', submodules=True)
```

Comment:
```[2021-09-22T08:59:23.543Z] Fetching tags only, you probably meant:

[2021-09-22T08:59:23.543Z]   git fetch --tags

[2021-09-22T08:59:23.543Z] error: pathspec '3.3.4' did not match any file(s) known to git.

[2021-09-22T08:59:23.543Z] ==> Error: FetchError: All fetchers failed
```
No tag yet?

Reply:
No. Apparently my genius is frightening sometimes. Thanks a lot for spotting it, I just created it.